tbot_contrib/gpio.py

```diff
@@ -0,0 +1,80 @@
+from tbot.machine import linux
+
+
+class Gpio:
+    def __init__(self, host: linux.LinuxShell, gpio_number: int):
+
+        self.host = host
+        self.gpio_number = gpio_number
+        self._gpio_sysclass_path = self.host.fsroot / "sys/class/gpio"
+        self._gpio_path = self._gpio_sysclass_path / f"gpio{self.gpio_number}"
+        self._export()
+        self._direction = self.get_direction()
+        self._active_low = self.get_active_low()
+
+    def _export(self) -> None:
+        if not self._gpio_path.is_dir():
+            (self._gpio_sysclass_path / "export").write_text(str(self.gpio_number))
+
+    def _is_in_direction(self) -> bool:
+        return self._direction == "in"
```

Comment:
I think this function is not needed; you can just use `self._direction == "in"` in the code below instead of calling `_is_in_direction()`.  This makes reading the code much easier as one does not need to cross-reference the functions as much.